src/main/java/Entities/ActionSpace2.java

```diff
@@ -0,0 +1,31 @@
+package Entities;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class ActionSpace2 extends Cell{
+    
+    private List<Card> cards;
+    private String type;
+
+    public ActionSpace2(HashMap<String, List<Card>> cards) {
+        String key = cards.keySet().iterator().next();
+        this.cards = cards.get(key);
+        this.type = key;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public List<Card> getCards() {
+        return cards;
+    }
+
+    @Override
+    public String performAction(Player player, Board board) {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+}
```

Comment:
I think a better naming convention can be used for this class, and on top of that change the name of the other action space class with respect to this one.